engines/director/detection_tables.h

```diff
@@ -9197,6 +9198,9 @@ static const DirectorGameDescription gameDescriptions[] = {
 	WINGAME2("babarcoins", "", "Start.exe",		"b198e9bf56c679747e4fd0972d5ba7b7", 2659300,
 							   "ba-01_A01.dxr", "cb38821091082aeb097e88551b791e05", 9946435, 1010),
 
+	// Actually Game use different Engine, Play.exe is something like an AutoRun.
+	WINGAME1_l("billibanniplay", "", "Play.exe", "t:6c95f5fa5fa9e107cc1e8a50491e262d", 3501155, Common::DE_DEU, 1010),
```

Comment:
Then it is absolutely incorrect. The .exe file to provide must be the projector file.

Reply:
Play.exe is the Autorun of the Game, The Autorun is using Director MX 2004, the Game itself uses an other Engine.

Reply:
Is there any functionality in this autorun? Also, play.exe is way too generic, a second file needs to be added to the entry

Reply:
It starts the Game, it starts the installer. Like an normal autorun

Reply:
I agree with @sev-: I don't see the point of adding an installer to the detection entries

Reply:
Should i close then the PR? We have only one cklasse entry which have this version number as I am aware of it was version 1.0 Juni/July 1993, maybe i should move the information to the comments instead?

Reply:
If only one version of the game was released, then there's no point to specify it in the detection entry - both in the comments and the entry itself